extensions/portuguese-primeira-liga/src/api/index.ts

```diff
@@ -0,0 +1,33 @@
+import { preferences, showToast, Toast } from "@raycast/api";
+import axios, { AxiosRequestConfig } from "axios";
+import { find } from "lodash";
+import { Table } from "../types";
+
+const apiKey = preferences.apiKey?.value as string;
```

Comment:
I actually had it like this before, but since in this case `Preferences` will also be marked as deprecated, I just reverted it. I guess this is still the preferable way right?